MoltenVK/MoltenVK/GPUObjects/MVKImage.mm

```diff
@@ -1763,6 +1765,13 @@ static void signalAndUnmarkAsTracked(const MVKSwapchainSignaler& signaler) {
 		}
 	}
 
+	if (!mvkIsAnyFlagEnabled(_imageView->_image->_mtlUsage, MTLTextureUsagePixelFormatView)) {
+		if (!enableSwizzling()) {
+			assert(0); // PFV should be enabled otherwise
+		}
+		return VK_SUCCESS;
+	}
+
```

Comment:
Use `if ( !_imageView->_image->hasPixelFormatView()) {` instead?  

Since the validation logic here is a bit complex, and tied to things happening in `MVKPixelFormats`, prefer to use `MVKAssert(enableSwizzling(), ...)` so it will log a description of the problem. 

Finally, is `enableSwizzling()` the correct test here, since it supports `fullImageViewSwizzle`, but your PFV test in `MVKPixelFormats` only supports `nativeTextureSwizzle`?

Reply:
> `MVKAssert(enableSwizzling(), ...)`

Does MVKAssert run the test regardless of whether its assertions have been disabled?  We need the side effects of the call too.

> Finally, is `enableSwizzling()` the correct test here

hasPixelFormatView will only be false if nativeTextureSwizzle is true, so enableSwizzling should be fine

(I didn't include fullImageViewSwizzle in the PFV check since it can be turned on/off at any time according to the documentation)

Reply:
> Does MVKAssert run the test regardless of whether its assertions have been disabled? We need the side effects of the call too.

From `MVKLogging.h`:

```
#define MVKAssert(test, fmt, ...)				\
do {											\
	bool isErr = !(test);						\
	MVKLogErrorIf(isErr, fmt, ##__VA_ARGS__);	\
	assert(!isErr || MVK_BLOCK_ASSERTIONS);		\
} while(0)
```

Reply:
Okay, it's now using MVKAssert